sphinxext/minio.py

```diff
@@ -263,6 +263,8 @@ class MinioObject(ObjectDescription):
     option_spec = {
```

Comment:
Feel free to ignore changes to this page. They are largely to correct or improve the directive/roles for better linking.